azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDaoImpl.java

```diff
@@ -44,31 +48,34 @@
   private final DatabaseOperator databaseOperator;
 
   static String INSERT_IMAGE_TYPE =
-      "insert into image_types ( name, description, active, deployable, created_by, modified_by )"
-          + " values (?, ?, ?, ?, ?, ?)";
+      "insert into image_types ( name, description, active, deployable, created_by, "
+          + "created_on, modified_by, modified_on )"
+          + " values (?, ?, ?, ?, ?, ?, ?, ?)";
   static String INSERT_IMAGE_OWNERSHIP =
-      "insert into image_ownerships ( type_id, owner, role, created_by, modified_by ) "
-          + "values (?, ?, ?, ?, ?)";
+      "insert into image_ownerships ( type_id, owner, role, created_by, created_on, modified_by, "
+          + "modified_on ) values (?, ?, ?, ?, ?, ?, ?)";
 
   @Inject
-  public ImageTypeDaoImpl(DatabaseOperator databaseOperator) {
+  public ImageTypeDaoImpl(final DatabaseOperator databaseOperator) {
     this.databaseOperator = databaseOperator;
   }
 
   @Override
-  public int createImageType(ImageType imageType) {
+  public int createImageType(final ImageType imageType) {
     final SQLTransaction<Integer> insertAndGetSpaceId = transOperator -> {
       // insert image type record
-      transOperator.update(INSERT_IMAGE_TYPE, imageType.getName(), imageType.getDescription(),
-          true, imageType.getDeployable().getName(), imageType.getCreatedBy(),
-          imageType.getModifiedBy());
-      int imageTypeId = Long.valueOf(transOperator.getLastInsertId()).intValue();
+      // Passing timestamp from the code base and can be formatted accordingly based on timezone
+      final Timestamp currentTimestamp = Timestamp.valueOf(LocalDateTime.now());
+      transOperator.update(INSERT_IMAGE_TYPE, imageType.getName().toLowerCase(),
```

Comment:
Yes exception handling needs to be improved and need to throw 400 response in case for database exception. Also we are using common dbutils for db operation and it does not throw Constraint Violation exception instead throws SqlException for duplicate name. Also throws error message with complete query. Probably we need to catch this kind of exception with sql error code and requires some special handling. Will improve the exception handling part in the next PR.